python/air/backend/xrt.py

```diff
@@ -46,23 +46,28 @@ class XRTBackend(AirBackend):
     def __init__(
         self,
         verbose=False,
-        experimental_passes=False,
         omit_while_true_loop=False,
         omit_pingpong=False,
+        lower_linalg_to_func=False,
+        air_loop_fusion=False,
+        runtime_loop_tiling_sizes: list[int] = [4, 4],
```

Comment:
We got rid of one option but grew some new ones :/

Reply:
Yeah... But the good thing is that they now more clearly specify what "experimental" features were being used. The `lower_linalg_to_func` is likely to be necessary to say if we want micro-kernel or direct code-generated (scalar, for now) code.

I wish the `air-loop-fusion` and `runtim-loop-tiling-sizes` can go eventually...